src/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
```

Comment:
Run the test with `npm test`.

You will see how tests are running.

This is just an information, nothing to change.